lti_consumer/lti_xblock.py

```diff
@@ -1085,6 +1085,19 @@ def lti_1p3_launch_callback(self, request, suffix=''):  # pylint: disable=unused
             # This is optional though
             lti_consumer.set_launch_presentation_claim('iframe')
 
+            # Set context claim
+            # This is optional
+            context_title = " | ".join([
```

Comment:
Nit: change to ` - `
```suggestion
            context_title = " - ".join([
```

Reply:
Done in next commit